kuma/javascript/src/logo.jsx

```diff
@@ -0,0 +1,20 @@
+//@flow
+import * as React from 'react';
```

Comment:
Curious, what is the benefit/difference between this and `import React from "react";`?

Reply:
Doing it that way imports flow types like React.Node that I use below. The alternative is to have one import for the React module and then a separate 'import type' line (or lines) for the types.

I tend to favor separate type imports, but this seems like it might be an idiomatic thing that we could just get in the habit of doing so we always have React.Node available... 

This seems like the sort of thing we'll want to have a consistent style for eventually.